Utils/CommandHandler.py

```diff
@@ -0,0 +1,55 @@
+import IrcCommands
+
+#   When new command is created in IrcCommands folder, do the following
+#     1. Add the import in IrcCommands/__init__.py
+#     2. Create a function here to call the function in IrcCommands
+#     3. Add the command trigger to CommandsList
+#     4. Add trigger:function to CommandSwitch
+def Handle(user, msg, ircClient):
+    CommandsList = [
+        "is listening",
+        "is playing",
+        "!with",
+        "!mods",
+        "!discord",
+        "!help"
+    ]
+    CommandSwitch = {
+        "is listening" : isListening,
+        "is playing" : isPlaying,
+        "!with" : with_,
+        "!mods" : mods,
+        "!discord" : discord,
+        "!help" : help_,
+        "default" : default
+    }
+    actualCommand = parseCommand(msg, CommandsList)
```

Comment:
these two vars can be de-deuplicated to make it a bit safer from bugs/easier to maintain

consider deleting `CommandsList`, and then simply change this line to:
`parseCommand(msg, CommandSwitch.keys())`